### PR TITLE
fix(currency-exchange-rate): Currency exchange rate calculation

### PR DESCRIPTION
**Pull request by abhi-markan**
## Introduction
Currency exchange for non GBP source currencies.

## Resolution
* Refactored currency inversion for non-GBP i.e. `USD -> GBP`
* Enhanced api tests.
* Improved documentation.

## Miscellaneous
Renamed PS title from source currency to GBP since they are calculated in GBP.

### CODE REVIEW

File: trade-finance-manager-ui/templates/case/facility/_macros/premium_schedule.njk [hunk 1]
```diff
@@ -84,7 +84,7 @@
             <tr class="govuk-table__row">
               <th scope="col" class="govuk-table__header govuk-!-width-one-quarter govuk-table__header--numeric govuk-body-s" data-cy="facilities-table-heading-facility-id">Payment</th>
               <th scope="col" class="govuk-table__header govuk-!-width-one-quarter govuk-table__header--numeric govuk-body-s" data-cy="facilities-table-heading-cover-end-date">Due date</th>
-              <th scope="col" class="govuk-table__header govuk-!-width-one-quarter govuk-table__header--numeric govuk-body-s" data-cy="facilities-table-heading-value-gbp">Amount to be paid to UKEF ({{ currencySymbol }})</th>
+              <th scope="col" class="govuk-table__header govuk-!-width-one-quarter govuk-table__header--numeric govuk-body-s" data-cy="facilities-table-heading-value-gbp">Amount to be paid to UKEF (GBP)</th>
```
Comment (IgnatG) [inline trade-finance-manager-ui/templates/case/facility/_macros/premium_schedule.njk:87]: Are all currencies are in GBP here? I assume {{ currency symbol }} could return other currencies

File: reference-data-proxy/api-tests/v1/currency-exchange-rate.api-test.ts
Comment (IgnatG) [inline reference-data-proxy/api-tests/v1/currency-exchange-rate.api-test.ts:1]: Can we add a test for RON to GBP and vice versa? USD and euro were working previously by default. JPY is a new currency we introduced

Comment (abhi-markan) [reply reference-data-proxy/api-tests/v1/currency-exchange-rate.api-test.ts:1]: Have added some additional tests cases.

**Review by IgnatG — APPROVED** [review]
LGTM

**Pull request merged**